openfe/tests/setup/test_atommapping.py

```diff
@@ -12,3 +14,18 @@ def test_atommapping_usage(simple_mapping):
 def test_atommapping_hash(simple_mapping, other_mapping):
     # these two mappings map the same molecules, but with a different mapping
     assert simple_mapping is not other_mapping
+
+
+def test_draw_mapping_cairo(tmpdir, simple_mapping):
+    with tmpdir.as_cwd():
+        simple_mapping.draw_to_file('test.png')
+        filed = pathlib.Path('test.png')
+        assert filed.exists()
```

Comment:
Asserting the contents of the files seems like it will become problematic quickly if the backend changes in any way, so I thought it might be best to just stick with "does file exist".